spec/index.bs

```diff
@@ -2907,25 +2911,27 @@ The {{GPUTextureUsage}} flags determine how a {{GPUTexture}} may be used after i
 
             **Returns:** {{GPUTexture}}
 
+            1. If |descriptor|.{{GPUTextureDescriptor/format}} is a {{GPUTextureFormat}} that requires a feature (see
+                [[#texture-format-caps]]), but |this|.{{GPUObjectBase/[[device]]}}.{{device/[[features]]}} does not
+                [=list/contain=] the feature, throw a {{TypeError}}.
```

Comment:
Something to do (in a totally separate PR) would be to go through all of the places in the API that take GPUTextureFormat and make sure they're all throwing TypeErrors for unknown ones. createView for example.